styles/global.css

```diff
@@ -43,3 +42,12 @@ input[type='number'] {
   -ms-overflow-style: none; /* IE and Edge */
   scrollbar-width: none; /* Firefox */
 }
+
+.OwnMusic
+{
+  -webkit-text-stroke: 1px #AC4444;
+  color: transparent;
+}
+.lineHeight-1{
+line-height: 1;
+}
```

Comment:
Why are you unable to do this with tailwind css???
![Screenshot 2024-10-07 at 4 58 00 PM](https://github.com/user-attachments/assets/c0f2d34b-a61e-4de5-b091-37b470389f6f)